packages/orchestrator-ui-components/src/hooks/DataFetchHooks.ts

```diff
@@ -1,34 +1,7 @@
-import { useContext } from 'react';
-import { useQuery } from 'react-query';
-
-import { OrchestratorConfigContext } from '@/contexts';
-import { GraphqlFilter, ProcessDetailResultRaw } from '@/types';
-
-import { useQueryWithFetch } from './useQueryWithFetch';
-import { useWfoSession } from './useWfoSession';
+import { GraphqlFilter } from '@/types';
 
 export type CacheNames = { [key: string]: string };
 
-export const useRawProcessDetails = (processId: string) => {
-    const { orchestratorApiBaseUrl } = useContext(OrchestratorConfigContext);
-    const url = `${orchestratorApiBaseUrl}/processes/${processId}`;
-    return useQueryWithFetch<ProcessDetailResultRaw, Record<string, never>>(
-        url,
-        {},
-        `RawProcessDetails-${processId}`,
-    );
-};
-
-export const useCacheNames = () => {
-    const { orchestratorApiBaseUrl } = useContext(OrchestratorConfigContext);
-    const url = `${orchestratorApiBaseUrl}/settings/cache-names`;
-    return useQueryWithFetch<CacheNames, Record<string, never>>(
-        url,
-        {},
-        'cacheNames',
-    );
-};
-
 export const filterDataByCriteria = <Type>(
```

Comment:
Because this is the only thing left in the file I think it shouldn't be in the hooks/DataFetchHooks file anymore. Maybe move it to a utils file?